aiogram/types/base.py

```diff
@@ -225,7 +227,10 @@ def __setitem__(self, key: str, value: typing.Any) -> None:
         if key in self.props:
             return self.props[key].set_value(self, value, self.conf.get('parent', None))
         self.values[key] = value
-        raise KeyError(key)
+
+        # Show and log a warning when Telegram silently adds new Fields
+        warnings.warn(f"Field '{key}' doesn't exist in {self.__class__}")
```

Comment:
Besides the canonical explanation in official documentation:

> 
> warnings.warn() in library code if the issue is avoidable and the client application should be modified to eliminate the warning
> 
> logging.warning() if there is nothing the client application can do about the situation, but the event should still be noted

Seems warnings is redundant here

Reply:
@andrew000, FYI

I'm waiting for removing this line and then this fixes will be published to the PyPi.